### PR TITLE
update_dcos-kafka_kafka-mesos.jar_to_latest

### PR DESCRIPTION
**Pull request by dmitrypekar**
Hey,

I've just updated dcos-kafka to 0.2.0 and kafka-mesos.jar to 0.9.2.0.
Please merge.

### CODE REVIEW

File: repo/packages/K/kafka/1/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "kafka",
+  "version": "0.9.2.0",
+  "scm": "https://github.com/mesos/kafka",
+  "maintainer": "support@mesosphere.io",
+  "description": "Apache Kafka running on top of Apache Mesos",
+  "framework": true,
+  "tags": ["mesosphere", "framework", "bigdata"],
```
Comment (kensipe) [inline repo/packages/K/kafka/1/package.json]: we need to remove mesosphere and framework from the tags.  this is used for user search and is intended to differentiate the services.  I would suggest:  "analytics", "queue", "pipeline" anything you imagine a user would search for that you would like kafka to be an answer for... or involved with.

File: repo/packages/K/kafka/1/marathon.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "id": "{{kafka.app.id}}",
```
Comment (kensipe) [inline repo/packages/K/kafka/1/marathon.json]: can we make this kafka.framework-name ?   the issue is if I install 2 versions of kafka, I am force to change to variables for this to work.  The framework-name and the app.id.

File: repo/packages/K/kafka/1/marathon.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  "cpus": {{kafka.app.cpus}},
+  "mem": {{kafka.app.mem}},
+  "instances": 1,
```
Comment (kensipe) [inline repo/packages/K/kafka/1/marathon.json]: this needs to be a variable `"instances" : {{kafka.app.instances}}`

**Comment by kensipe** [comment]
The command-line fails on uninstall... details below:

```
dcos package uninstall kafka
Error while fetching [http://ken-69059-elasticl-ce2322x3nt4a-1645660120.us-west-2.elb.amazonaws.com/mesos/master/shutdown]: HTTP 404: Not Found
```

**Comment by kensipe** [comment]
@dmitrypekar sorry for the delay... there looks to be a few things to resolve.

**Comment by kensipe** [comment]
while you are at it... either rebase or remove the index.json.gz file

File: repo/packages/K/kafka/1/package.json (hunk 1 above)
Comment (dmitrypekar) [inline repo/packages/K/kafka/1/package.json]: Fixed.

File: repo/packages/K/kafka/1/marathon.json (hunk 2 above)
Comment (dmitrypekar) [inline repo/packages/K/kafka/1/marathon.json]: Fixed.

File: repo/packages/K/kafka/1/marathon.json (hunk 3 above)
Comment (dmitrypekar) [inline repo/packages/K/kafka/1/marathon.json]: Fixed.

**Comment by dmitrypekar** [comment]
@kensipe 
I've rebased PR and fixed following items:
- removed app.id, used framework-name instead;
- added app.instances config param;
- fixed tags;
- retested using local mesos cluster and dcos cli;

Please merge.

**Comment by kensipe** [comment]
awesome!  thx... LGTM merging!

**Pull request merged**